src/app.js

```diff
@@ -14,6 +14,10 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.set('knex', knex);
 
 var routes = require('./routes')(app);
```

Comment:
Should we get rid of this? It could be useful if we have routes that don't fall into users, projects, activities, and checkins, but the only one I can think of would be the / endpoint, which *should* be served as a static file anyway.